CONTRIBUTING.md

```diff
@@ -1,17 +1,183 @@
-# Contributing to LLVM
+# Contributing to the Arm Toolchain
 
-Thank you for your interest in contributing to LLVM! There are many ways to
-contribute, and we appreciate all contributions.
+The Arm Toolchain repository is a fork of the *llvm-project*, with
+build scripts to produce Arm Toolchain for Embedded and Arm Toolchain
+for Linux.
 
-To get started with contributing, please take a look at the
-[Contributing to LLVM](https://llvm.org/docs/Contributing.html) guide. It
-describes how to get involved, raise issues and submit patches.
+Arm Toolchain for Embedded integrates *llvm-project* with the
+*picolibc* C-library or the *newlib* C-library via an overlay package.
 
-## Getting in touch
+Contributions are welcome that improve the project in areas including
+but not limited to:
+* Integration of upstream components.
+* Packaging.
+* Testing of the above.
+* Documentation.
 
-Join the [LLVM Discourse forums](https://discourse.llvm.org/), [Discord
-chat](https://discord.gg/xS7Z362), or #llvm IRC channel on
-[OFTC](https://oftc.net/).
+The Arm Toolchain repository is synchronized with the upstream
+*llvm-project* including the main and release branches. Arm Toolchain
+for Embedded aims to stay close to the head of the *picolibc* main
+branch. The newlib overlay package aims to track the most recent
```

Comment:
I suggest we stick to a consistent formatting of project names. In some places they are emphasized with stars, in others they aren't. For instance in this line, newlib does not have emphasis.